packages/graphback-cli/src/helpers/generate.ts

```diff
@@ -23,73 +25,68 @@ followed by ${chalk.cyan(`${cliName}db`)} to create database.
   try {
-    const models = new GlobSync('model/*.graphql', { cwd: process.cwd() })
+    const { folders, graphqlCRUD, db: { database }, client } = configInstance.config;
+
+    const models = new GlobSync(`${folders.model}/*.graphql`)
 
     if (models.found.length === 0) {
-      logError(`No graphql file found inside ${process.cwd()}/model folder.`)
+      logError(`No graphql file found inside ${folders.model} folder.`)
       process.exit(0)
     }
-    const configPath = `${process.cwd()}/config.json`
-
-    const { database, generation, client } = JSON.parse(readFileSync(configPath, "utf8"))
 
-    const path: string = process.cwd()
-    const schemaText: string = models.found.map((m: string) => readFileSync(`${path}/${m}`, 'utf8')).join('\n')
+    const schemaText: string = models.found.map((m: string) => readFileSync(`/${m}`, 'utf8')).join('\n')
 
-    const outputSchemaPath: string = `${process.cwd()}/src/schema/generated.ts`
-    const outputResolverPath: string = `${process.cwd()}/src/resolvers`
+    const pathForSchema: string = join(process.cwd(), folders.schema)
+    const outputSchemaPath: string = `${pathForSchema}/generated.ts`
 
-    const backend: GraphQLBackendCreator = new GraphQLBackendCreator(schemaText, generation)
+    const customResolvers: string = join(process.cwd(), folders.customResolvers, `custom`)
+    const generatedResolvers: string = join(process.cwd(), folders.generatedResolvers, `generated`)
 
+    const backend: GraphQLBackendCreator = new GraphQLBackendCreator(schemaText, graphqlCRUD)
     const generated: IGraphQLBackend = await backend.createBackend(database)
-    
-    let generatedClient
-    let clientPath: string
-    if(client) {
-      generatedClient = await backend.createClient()
-      clientPath = `${process.cwd()}/client`
-      if(!existsSync(clientPath)) {
-        mkdirSync(clientPath)
-      }
-    }
 
-    writeFileSync(outputSchemaPath, generated.schema)
-
-    writeFileSync(`${outputResolverPath}/index.ts`, generated.resolvers.index)
-    
-    if(!existsSync(`${outputResolverPath}/custom`)) {
-      mkdirSync(`${outputResolverPath}/custom`)
-    }
-
-    if(!existsSync(`${outputResolverPath}/generated`)) {
-      mkdirSync(`${outputResolverPath}/generated`)
-    }
+    checkAndCreateFolders(pathForSchema, customResolvers, generatedResolvers);
 
     generated.resolvers.custom.forEach((output: OutputResolver) => {
-      if(!existsSync(`${outputResolverPath}/custom/${output.name}.ts`) || output.name === 'index') {
-        writeFileSync(`${outputResolverPath}/custom/${output.name}.ts`, output.output)
+      if (!existsSync(`${customResolvers}/${output.name}.ts`) || output.name === 'index') {
+        writeFileSync(`${customResolvers}/${output.name}.ts`, output.output)
       }
     })
 
-    const resolverFiles = readdirSync(`${outputResolverPath}/generated`)
-    resolverFiles.forEach((file: string) => unlinkSync(`${outputResolverPath}/generated/${file}`))
```

Comment:
This was for the case that the user changes his types. So there would be no need for a resolver file from the older model.

Reply:
Removing people's code - that they could have made modifications on is risky business. Especially that changes might not be committed. 
So people will be able to manually remove them later and there is no need for them to exist due to the fact that they will not be required in index. It is tradeoff for now. 

More work will be done in #340